frontend/src/components/bs-space/bs-space.tsx

```diff
@@ -0,0 +1,91 @@
+import { IconName } from 'common/enums/enums';
+import { IconButton } from 'components/common/common';
+import { useAppDispatch, useEffect, useParams } from 'hooks/hooks';
+import { ObjectsTable } from './components/components';
+import React, { FC } from 'react';
+import styles from './styles.module.scss';
+import { BSSpace as BSSpaceActions } from 'store/actions';
+
+const BSSpace: FC = () => {
+  const dispatch = useAppDispatch();
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    dispatch(
+      BSSpaceActions.loadObjects({
+        filter: {
+          from: 0,
+          count: 5,
+        },
+        params: {
+          id: id as string,
+        },
+      }),
+    );
+  }, [dispatch]);
+
+  const handleObjectsReload = (): void => {
+    dispatch(
+      BSSpaceActions.loadObjects({
+        filter: {
+          from: 0,
+          count: 5,
+        },
+        params: {
+          id: id as string,
+        },
+      }),
+    );
+  };
+
+  const handleObjectDownload = (objectId: string, filename: string): void => {
+    dispatch(
+      BSSpaceActions.downloadObject({
+        filename,
+        spaceId: id as string,
+        objectId,
+      }),
+    );
+  };
+
+  const handleUploading = (e: React.FormEvent<HTMLInputElement>): void => {
+    const files = e.currentTarget.files;
+    const formData = new FormData();
+    if (files) {
+      formData.append('file', files[0]);
+    }
```

Comment:
```suggestion
    const [file] = e.currentTarget.files;

    if (!hasFile) {
      return
    }
    
        const formData = new FormData();
```

Reply:
![image](https://user-images.githubusercontent.com/86227025/160290106-72c41ee9-08a0-4e86-923d-9bea9cbfd79f.png)
Why do I need a redundant variable hasFile(all js developers know that (!files) in if block converts to the boolean))?
And what is the problem of (!files)?)
![image](https://user-images.githubusercontent.com/86227025/160292099-34563658-6e64-4af5-a23f-30e502fdec53.png)

Reply:
> all js developers know tha
I don't know 🤔